src/engines/kotor/gui/widgets/kotorwidget.cpp

```diff
@@ -121,6 +125,13 @@ void KotORWidget::setPosition(float x, float y, float z) {
 
 		_text->setPosition(tX - oX + x, tY - oY + y, tZ - oZ + z);
 	}
+
+    if (_border) {
+        float bX, bY, bZ;
+		_border->getPosition(bX, bY, bZ);
+
+        _border->setPosition(x, y, z);
```

Comment:
_border->getPosition() has a side-effect, a comment would be in order. If it does not have a side-effect (i.e., it is pure), _border->getPosition() and bX, bY, and bZ are superfluous.